packages/dx-react-grid/src/plugins/table-fixed-columns.jsx

```diff
@@ -62,19 +66,21 @@ export class TableFixedColumns extends React.PureComponent {
       >
         <Getter name="tableHeaderRows" computed={tableHeaderRowsComputed} />
         <Getter name="tableColumns" computed={tableColumnsComputed} />
+        <Getter name="tableHeaderColumnChains" computed={tableHeaderColumnChainsComputed} />
         <Template
           name="tableCell"
           predicate={({ tableColumn }) => !!tableColumn.fixed}
         >
           {params => (
             <TemplateConnector>
-              {({ tableColumns }) => {
+              {({ tableColumns, tableHeaderColumnChains }) => {
```

Comment:
Currently, this plugin is dependent on the `TableBandHeader` plugin. We should change `pluginDependencies` constant.

```js
const pluginDependencies = [
   { name: 'Table' },
   { name: 'TableBandHeader', optional: true },
 ];
```

Reply:
I think we should fix it in separate PR, here's a [task](https://trello.com/c/NTirap7m/1285-grid-other-header-plugins-should-be-defined-as-optional-dependency-for-fixed-columns-plugin) in trello